api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -122,15 +134,32 @@ message DeleteAccessListRequest {
 // DeleteAllAccessListsRequest is the request for deleting all access lists.
 message DeleteAllAccessListsRequest {}
 
-// GetAccessListsToReviewRequest is the request for getting access lists that the current user needs to review.
+// GetAccessListsToReviewRequest is the request for getting access lists that
+// the current user needs to review.
 message GetAccessListsToReviewRequest {}
 
-// GetAccessListsToReviewResponse is the response for getting access lists that the current user needs to review.
+// GetAccessListsToReviewResponse is the response for getting access lists that
+// the current user needs to review.
 message GetAccessListsToReviewResponse {
   repeated AccessList access_lists = 1;
 }
 
-// ListAccessListMembersRequest is the request for getting paginated access list members.
+// CountAccessListMembersRequest is the request for counting access list
+// members.
+message CountAccessListMembersRequest {
+  // access_list_name is the name of the access list to retrieve.
+  string access_list_name = 1;
+}
+
+// CountAccessListMembersResponse is the response for counting access list
+// members.
+message CountAccessListMembersResponse {
+  // count is the number of access list members in the access list.
+  int32 count = 1;
```

Comment:
Why a signed value for a non-negative integer? Why 32 bits?

Reply:
I chose 32 bits because i figured it'd be enough to house any possible count here while giving sizeable headroom. It should be unsigned, though.